src/antelope/config/index.ts

```diff
@@ -3,6 +3,12 @@ import { App } from 'vue';
 import { getAntelope } from '..';
 
 export class AntelopeConfig {
+    // indexer health threshold --
+    private __indexer_health_threshold = 10;
+
+    // indexer health check interval --
+    private __indexer_health_check_interval = 1000 * 60 * 5;
```

Comment:
can you add a comment here for both of these numbers indicating what unit of time they represent?